src/sentry/constants.py

```diff
@@ -790,6 +790,19 @@ class InsightModules(Enum):
     "*/readyz",
     "*/ping",
     "*/up",
+    # Same as above with trailing slash. Relay's pattern matcher cannot handle
+    # optional trailing slashes.
+    "*/health/",
+    "*/healthy/",
+    "*/healthz/",
+    "*/_health/",
+    r"*/\[_health\]/",
+    "*/live/",
+    "*/livez/",
+    "*/ready/",
+    "*/readyz/",
+    "*/ping/",
+    "*/up/",
```

Comment:
Should we do this in Code instead? Automatically duplicate all entries with a slash?

Reply:
Will wait for https://github.com/getsentry/relay/pull/4277 instead.

Reply:
That PR has been superseded by https://github.com/getsentry/relay/pull/4283, which is now merged.